REFERENCE.md

```diff
@@ -0,0 +1,90 @@
+---
+RFC: <PR number>
+Status: <Proposed>
+---
+
+# Title (Required)
+
+## Abstract (Required)
+
+A few sentences describing the feature.
+
+## Motivation (Required)
+
+What the feature solves and why the existing functionality is not enough.
+
+## Design considerations (Required)
+
+A description of the design constraints and requirements for the proposal. Comparisons with similar features in other projects.
+
+## Specification (Required)
+
+A more detailed description of the feature, including why certain details in the design have been chosen.
+
+### Commands (Optional)
+
+If any new commands are introduced:
+
+1. Command name
+   - **Request**
+   - **Response**
+
+### Authentication and Authorization (Optional)
+
+If there are any changes around introducing new ACL command/categories for user access control.
+
+### Append-only file (Optional)
```

Comment:
I agree there might be different considerations about AOF and RDB. but maybe for the sake of simplicity we can set a section name for 'Persistency' which will capture all of these?

Reply:
Moving them next to each other.